arduino/temp-netsender/temp-netsender.ino

```diff
@@ -105,6 +110,17 @@ int tempReader(NetSender::Pin *pin) {
       pin->value = 10 * (ff + ZERO_CELSIUS);
       break;
     }
+  case 70: // TSL Lux
+    tsl.getEvent(&event);
+    if ((event.light == 0) |
```

Comment:
Shouldn't we also check for <= 0, which doesn't make sense for lux ? We need to keep pin values above 0 given that -1 indicates an error or null value.